charts/cronjob/templates/cronjob.yaml

```diff
@@ -19,6 +19,10 @@ spec:
           labels:
             app: {{ template "cronjob.name" . }}
             release: {{ .Release.Name }}
+          annotations:
+            # Force redeployment when external resource secrets.yaml changes
+            # Related issue: https://github.com/commercetools/k8s-charts/issues/14
```

Comment:
No need for this comment IMO. the issue is referenced from the PR description

Reply:
Yes, but like this you don't have to go to the issue. It's only 1 sentence description. If you need more details, go there.

Reply:
Are we going to put issue links to all our bug fixes?